indexer/snapshot_full_node_ap_northeast_1.tf

```diff
@@ -25,9 +25,9 @@ module "full_node_snapshot_ap_northeast_1" {
   datadog_api_key = var.datadog_api_key
   dd_site         = var.dd_site
 
-  # in public testnet, use the validator image which contains the snapshot script.
+  # in public testnet, use the node image which contains the snapshot script.
   # in dev environments, we build separate images.
-  ecr_repository_url = contains(["testnet", "testnet1", "testnet2"], var.environment) ? data.aws_ecr_repository.validator.repository_url : data.aws_ecr_repository.snapshot_validator.repository_url
+  ecr_repository_url = contains(["testnet", "testnet1", "testnet2"], var.environment) ? local.node_ecr_repository_url : local.snapshot_node_ecr_repository_url
```

Comment:
should mainnet have the same behavior as our testnet or dev/staging for the snapshot node?

Reply:
It's not currently deployed so that's an unknown right now.

Reply:
why do we have a separate ECR repo for the snapshot node if we can just use the indexer full node ECR repo in testnet?

cc: @dydxwill

Reply:
I haven't finished changing the deployment of the images for the env, so it's an unknown.